### PR TITLE
add iPhone 6 offsets

### PR DESCRIPTION
**Pull request by drake90001**

### CODE REVIEW

File: yalu102/offsets.c [hunk 1]
```diff
@@ -34,9 +34,13 @@ void init_offsets() {
...
         allproc_offset = 0x5a8438;
-    procoff = 0x360;
-    rootvnode_offset = 0x5ae0b8;
-    } else {
+        procoff = 0x360; // iphone 6s
+        rootvnode_offset = 0x5ae0b8;
+    } else if (strcmp(u.version, "Darwin Kernel Version 16.3.0: Tue Nov 29 21:40:08 PST 2016; root:xnu-3789.32.1~4/RELEASE_ARM64_T7000") == 0) {
+        allproc_offset = 0x5b8468;
+        procoff = 0x360; // iphone 6, credit to @Andywiik
+        rootvnode = 0x5be0b8;
```
Comment (liudayu) [inline yalu102/offsets.c]: you mean "rootvnode_offset" at line 42?

Comment (drake90001) [inline yalu102/offsets.c]: Thank you

**Comment by nullpixel1** [comment]
It's panicking according to @applebetas01

**Comment by vmunich** [comment]
This is crashing on the 6 Plus n56 board.

```
sysname: Darwin
nodename: iPhone-6Plus
release: 16.3.0
version: Darwin Kernel Version 16.3.0: Tue Nov 29 21:40:08 PST 2016; root:xnu-3789.32.1~4/RELEASE_ARM64_T7000
machine: iPhone7,1
2017-01-26 14:25:09.042389 yalu102[224:4500] found corruption 37c03
2017-01-26 14:25:13.011439 yalu102[224:4500] found kernel text at fffffff017204000
2017-01-26 14:25:13.013557 yalu102[224:4500] got tfp0 -> 37d03
2017-01-26 14:25:13.015376 yalu102[224:4500] found procs at fffffff11c483830
2017-01-26 14:25:13.019576 yalu102[224:4500] seg: __TEXT
2017-01-26 14:25:13.019649 yalu102[224:4500] seg: __DATA_CONST
2017-01-26 14:25:13.019685 yalu102[224:4500] seg: __TEXT_EXEC
2017-01-26 14:25:13.019718 yalu102[224:4500] seg: __KLD
2017-01-26 14:25:13.019750 yalu102[224:4500] seg: __LAST
2017-01-26 14:25:13.019782 yalu102[224:4500] seg: __DATA
2017-01-26 14:25:13.019867 yalu102[224:4500] seg: __PRELINK_TEXT
2017-01-26 14:25:13.019921 yalu102[224:4500] seg: __PLK_TEXT_EXEC
2017-01-26 14:25:13.019954 yalu102[224:4500] seg: __PRELINK_DATA
2017-01-26 14:25:13.019986 yalu102[224:4500] seg: __PLK_DATA_CONST
2017-01-26 14:25:13.020018 yalu102[224:4500] seg: __PLK_LINKEDIT
2017-01-26 14:25:13.020049 yalu102[224:4500] seg: __PRELINK_INFO
2017-01-26 14:25:13.020160 yalu102[224:4500] seg: __LINKEDIT
2017-01-26 14:25:13.020220 yalu102[224:4500] fffffff0162cc000 - fffffff017a94000
2017-01-26 14:25:13.077948 yalu102[224:4500] ffffffff03e38000
2017-01-26 14:25:13.125432 yalu102[224:4500] fffffff0172715a0
2017-01-26 14:25:13.164286 yalu102[224:4500] pmap: fffffff017720850
2017-01-26 14:25:13.203155 yalu102[224:4500] got phys at 0 for virt fffffff000801000
2017-01-26 14:25:13.203270 yalu102[224:4500] found cpu 0
2017-01-26 14:25:13.203301 yalu102[224:4500] found physz: fffffff017293000
2017-01-26 14:25:13.203335 yalu102[224:4500] found cpu 1
2017-01-26 14:25:13.203362 yalu102[224:4500] found physz: fffffff017293000
2017-01-26 14:25:13.203427 yalu102[224:4500] fffffff017393574 - fffffff0173933a8
2017-01-26 14:25:13.204858 yalu102[224:4500] ttbr0: 802ab9000 fffffff017271590
2017-01-26 14:25:13.207364 yalu102[224:4500] got a cpacr
```

```
_allproc: 0x5b8468
_kernproc: 0x5be0e0
$ nm kernelcachen56.decompressed | grep rootvnode
fffffff0075c20b8 S _rootvnode
fffffff0071e1034 S _vfs_rootvnode
```

**Comment by vmunich** [comment]
I just read about how it doesn't support 4k devices yet. Sorry about that.

**Pull request merged**